test/integration/sequelize/deferrable.test.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+/* jshint -W030 */
+/* jshint -W110 */
+var _ = require('lodash')
+  , chai = require('chai')
+  , expect = chai.expect
+  , assert = chai.assert
+  , Support = require(__dirname + '/../support')
+  , dialect = Support.getTestDialect()
+  , Sequelize = require(__dirname + '/../../../index')
+  , config = require(__dirname + '/../../config/config')
+  ;
+
+describe(Support.getTestDialectTeaser('Sequelize'), function() {
+  if (dialect !== 'postgres') {
+    return;
+  }
```

Comment:
This will print the teaser for all other dialects, so it would be nice to wrap the teaser in the if instead.